src/__tests__/organization.tests.ts

```diff
@@ -0,0 +1,319 @@
+        //Assert
+        expect(organizationResponse.status).toBe(200);
+        console.log(organizationResponse);
+        expect(organizationResponse.data).toBeGreaterThan(0);
+  
+      });
+      });
+
+      describe("Validation", () => {
+        it("When updating new oraganization with empty name, Expect to return HTTP Status 400", async () => {
+          //Arrange
+          let oraganizationApi = new OrganizationApi(configurationWithAuthHeader);
+          //Act
+          let payload = TdeiObjectFaker.getOrganization();
+          payload.org_name = '';
+    
+          const oraganizationResponse = oraganizationApi.updateOrganization(payload);
+    
+          //Assert
+          await expect(oraganizationResponse).rejects.toMatchObject({response: { status: 400 } });
+        });
+    
+        it("When updating new oraganization with empty phone, Expect to return HTTP Status 400", async () => {
+          //Arrange
+          let oraganizationApi = new OrganizationApi(configurationWithAuthHeader);
+          //Act
+          let payload = TdeiObjectFaker.getOrganization();
+          payload.phone = '';
+    
+          const oraganizationResponse = oraganizationApi.updateOrganization(payload);
+    
+          //Assert
+          await expect(oraganizationResponse).rejects.toMatchObject({response: { status: 400 } });
+        });
+    
+        it("When updating new oraganization with empty address, Expect to return HTTP Status 400", async () => {
+          //Arrange
+          let oraganizationApi = new OrganizationApi(configurationWithAuthHeader);
+          //Act
+          let payload = TdeiObjectFaker.getOrganization();
+          payload.address = '';
+    
+          const oraganizationResponse = oraganizationApi.updateOrganization(payload);
+    
+          //Assert
+          await expect(oraganizationResponse).rejects.toMatchObject({response: { status: 400 } });
+        });
+    
+        it('When updating new organization with invalid polygon, Expect to return HTTP Status 400', async () => {
+          //Arrange
+          let oraganizationApi = new OrganizationApi(configurationWithAuthHeader);
+          //Act
+          let payload = TdeiObjectFaker.getOrganization();
+          payload.polygon = TdeiObjectFaker.getInvalidPolygon();
+    
+          const oraganizationResponse = oraganizationApi.updateOrganization(payload);
+    
+          //Assert
+          await expect(oraganizationResponse).rejects.toMatchObject({ response: { status: 400 } });
+         });
+        });
+      });
+
+      describe('Get Station', () => {
+        describe('Auth', () => {
+          it("When no auth token provided, Expect to return HTTP status 401", async () => {
+            //Arrange
+            let oraganizationApi = new OrganizationApi(configurationWithoutAuthHeader);
+            //Act
+            const request = async () => {
```

Comment:
Change the syntax as done previous cases

Reply:
fixed